### ISSUE TITLE
[BUG] Error installing private GitLab repositories in v7

### ISSUE THREAD

**Issue by rsispal** [npm/cli#1963]
### Current Behavior:
When running `npm install` on projects that utilise private GitLab repositories as dependencies (authenticated via SSH key on my machine), I receive the following error:

```
npm ERR! Wrong spec type (git) for RemoteFetcher. Supported types: remote
npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/-/.npm/_logs/2020-10-14T23_36_32_883Z-debug.log
```

And here is the error excerpt from the log file:
```
2601 verbose stack TypeError: Wrong spec type (git) for RemoteFetcher. Supported types: remote
2601 verbose stack     at RemoteFetcher.[_assertType] (/usr/local/lib/node_modules/npm/node_modules/pacote/lib/fetcher.js:219:13)
2601 verbose stack     at new FetcherBase (/usr/local/lib/node_modules/npm/node_modules/pacote/lib/fetcher.js:59:22)
2601 verbose stack     at new RemoteFetcher (/usr/local/lib/node_modules/npm/node_modules/pacote/lib/remote.js:14:5)
2601 verbose stack     at /usr/local/lib/node_modules/npm/node_modules/pacote/lib/git.js:208:16
2601 verbose stack     at /usr/local/lib/node_modules/npm/node_modules/cacache/lib/util/disposer.js:26:43
2602 verbose cwd /Users/-/-
2603 verbose Darwin 19.6.0
2604 verbose argv "/usr/local/bin/node" "/usr/local/bin/npm" "i"
2605 verbose node v13.9.0
2606 verbose npm  v7.0.0
2607 error Wrong spec type (git) for RemoteFetcher. Supported types: remote
2608 verbose exit 1
```

As it stands, I am unable to utilise NPM v7 with private GitLab repositories. I dont think I've missed anything as there are no errors when using npm 6.14.8.

### Expected Behavior:
Each of these projects which use GitLab private repos (_authenticated on my machine via SSH key, not HTTPS_) as dependencies used to install and the project would run as expected.

### Steps To Reproduce:

1. Create a private repository on GitLab
2. Ensure your machine can access/clone private repositories from GitLab via SSH
3. Initialise an NPM project
4. Install your dependency via `npm i --save "git+ssh://git@gitlab.com:path/to/repo/myreponame.git#master"`
5. See the above error

### Environment:

- OS: MacOS 10.15.6 Catalina
- Node: 13.9.0
- npm: 7.0.0

**Comment by nlf** [comment]
hi there! we released 7.0.1 last night, if you update to `npm i -g npm@next-7` it should resolve this!

let us know how it goes!

**Comment by rsispal** [comment]
Hi @nlf, thank you for informing me about this very rapid fix! I can confirm the problem is resolved, so I will close this issue. Thank you very much

**Comment by codejedi365** [comment]
We should re-open this. The same error has re-emerged in `NPM@v7.7.0`.  But this also affects public repositories.  Running this same command in `NPM@v6.10.2` it installs successfully.

I tried to run the command
```sh
npm install --save gitlab:gitlab-org/security-products/security-report-schemas#semver:^14.0.1
```
and received the same error like above:
```log
npm ERR! Wrong spec type (git) for RemoteFetcher. Supported types: remote
```
What is also odd is that it seems to access and read the repository to install dependencies but then fails towards the end of the log. Here is the full log:
```log
0 verbose cli [
0 verbose cli   '/Users/codejedi365/.nvm/versions/node/v14.16.0/bin/node',
0 verbose cli   '/Users/codejedi365/.nvm/versions/node/v14.16.0/bin/npm',
0 verbose cli   'install',
0 verbose cli   '--save',
0 verbose cli   'gitlab:gitlab-org/security-products/security-report-schemas#semver:^14.0.1'
0 verbose cli ]
1 info using npm@7.7.0
2 info using node@v14.16.0
3 timing npm:load:whichnode Completed in 1ms
4 timing config:load:defaults Completed in 2ms
5 timing config:load:file:/Users/codejedi365/.nvm/versions/node/v14.16.0/lib/node_modules/npm/npmrc Completed in 1ms
6 timing config:load:builtin Completed in 1ms
7 timing config:load:cli Completed in 2ms
8 timing config:load:env Completed in 0ms
9 timing config:load:file:/Users/codejedi365/Development/workbench/workspace/gitlab-npm-audit-parser/.npmrc Completed in 0ms
10 timing config:load:project Completed in 1ms
11 timing config:load:file:/Users/codejedi365/.npmrc Completed in 0ms
12 timing config:load:user Completed in 0ms
13 timing config:load:file:/Users/codejedi365/.nvm/versions/node/v14.16.0/etc/npmrc Completed in 0ms
14 timing config:load:global Completed in 0ms
15 timing config:load:validate Completed in 0ms
16 timing config:load:setEnvs Completed in 1ms
17 timing config:load Completed in 7ms
18 timing npm:load:configload Completed in 7ms
19 timing npm:load:setTitle Completed in 0ms
20 timing npm:load:setupLog Completed in 2ms
21 timing npm:load:cleanupLog Completed in 1ms
22 timing npm:load:configScope Completed in 0ms
23 timing npm:load:projectScope Completed in 0ms
24 timing npm:load Completed in 12ms
25 timing config:load:flatten Completed in 2ms
26 timing arborist:ctor Completed in 1ms
27 timing idealTree:init Completed in 21ms
28 timing arborist:ctor Completed in 0ms
29 timing idealTree:userRequests Completed in 1902ms
30 silly idealTree buildDeps
31 silly fetch manifest schema-merge@gitlab:gitlab-org/security-products/security-report-schemas#semver:^14.0.1
32 silly placeDep ROOT schema-merge@1.0.0 OK for: @elpete/gitlab-npm-audit-parser@1.1.0 want: gitlab:gitlab-org/security-products/security-report-schemas#semver:^14.0.1
33 silly fetch manifest esm@^3.2.25
34 silly fetch manifest jasmine@^3.6.3
35 silly fetch manifest json-schema-merge-allof@^0.7.0
36 silly fetch manifest json-schema-ref-parser@^8.0.0
37 silly fetch manifest jsonschema@^1.4.0
38 silly fetch manifest jsonvalidator@^0.1.1
39 http fetch GET 304 https://registry.npmjs.org/jsonschema 196ms (from cache)
40 http fetch GET 304 https://registry.npmjs.org/json-schema-merge-allof 198ms (from cache)
41 http fetch GET 304 https://registry.npmjs.org/jasmine 200ms (from cache)
42 http fetch GET 304 https://registry.npmjs.org/jsonvalidator 198ms (from cache)
43 http fetch GET 304 https://registry.npmjs.org/json-schema-ref-parser 204ms (from cache)
44 http fetch GET 304 https://registry.npmjs.org/esm 213ms (from cache)
45 timing idealTree:#root Completed in 1464ms
46 silly placeDep ROOT esm@3.2.25 OK for: schema-merge@1.0.0 want: ^3.2.25
47 silly placeDep ROOT jasmine@3.7.0 OK for: schema-merge@1.0.0 want: ^3.6.3
48 silly placeDep ROOT json-schema-merge-allof@0.7.0 OK for: schema-merge@1.0.0 want: ^0.7.0
49 silly placeDep ROOT json-schema-ref-parser@8.0.0 OK for: schema-merge@1.0.0 want: ^8.0.0
50 silly placeDep ROOT jsonschema@1.4.0 OK for: schema-merge@1.0.0 want: ^1.4.0
51 silly placeDep ROOT jsonvalidator@0.1.1 OK for: schema-merge@1.0.0 want: ^0.1.1
52 silly fetch manifest glob@^7.1.6
53 silly fetch manifest jasmine-core@~3.7.0
54 silly fetch manifest compute-lcm@^1.1.0
55 silly fetch manifest json-schema-compare@^0.2.2
56 silly fetch manifest lodash@^4.17.4
57 silly fetch manifest @apidevtools/json-schema-ref-parser@8.0.0
58 silly fetch manifest ajv@^4.11.3
59 silly fetch manifest debug@^2.6.1
60 http fetch GET 304 https://registry.npmjs.org/compute-lcm 64ms (from cache)
61 http fetch GET 304 https://registry.npmjs.org/jasmine-core 66ms (from cache)
62 http fetch GET 304 https://registry.npmjs.org/json-schema-compare 65ms (from cache)
63 http fetch GET 304 https://registry.npmjs.org/glob 81ms (from cache)
64 http fetch GET 304 https://registry.npmjs.org/lodash 78ms (from cache)
65 http fetch GET 304 https://registry.npmjs.org/@apidevtools%2fjson-schema-ref-parser 126ms (from cache)
66 http fetch GET 304 https://registry.npmjs.org/debug 126ms (from cache)
67 http fetch GET 304 https://registry.npmjs.org/ajv 129ms (from cache)
68 timing idealTree:node_modules/schema-merge Completed in 149ms
69 timing idealTree:node_modules/esm Completed in 0ms
70 silly placeDep ROOT glob@7.1.7 OK for: jasmine@3.7.0 want: ^7.1.6
71 silly placeDep ROOT jasmine-core@3.7.1 OK for: jasmine@3.7.0 want: ~3.7.0
72 silly fetch manifest fs.realpath@^1.0.0
73 silly fetch manifest inflight@^1.0.4
74 silly fetch manifest inherits@2
75 silly fetch manifest minimatch@^3.0.4
76 silly fetch manifest once@^1.3.0
77 silly fetch manifest path-is-absolute@^1.0.0
78 http fetch GET 304 https://registry.npmjs.org/inherits 77ms (from cache)
79 http fetch GET 304 https://registry.npmjs.org/inflight 77ms (from cache)
80 http fetch GET 304 https://registry.npmjs.org/once 91ms (from cache)
81 http fetch GET 304 https://registry.npmjs.org/minimatch 96ms (from cache)
82 http fetch GET 304 https://registry.npmjs.org/fs.realpath 99ms (from cache)
83 http fetch GET 304 https://registry.npmjs.org/path-is-absolute 94ms (from cache)
84 timing idealTree:node_modules/jasmine Completed in 103ms
85 silly placeDep ROOT fs.realpath@1.0.0 OK for: glob@7.1.7 want: ^1.0.0
86 silly placeDep ROOT inflight@1.0.6 OK for: glob@7.1.7 want: ^1.0.4
87 silly placeDep ROOT inherits@2.0.4 OK for: glob@7.1.7 want: 2
88 silly placeDep ROOT minimatch@3.0.4 OK for: glob@7.1.7 want: ^3.0.4
89 silly placeDep ROOT once@1.4.0 OK for: glob@7.1.7 want: ^1.3.0
90 silly placeDep ROOT path-is-absolute@1.0.1 OK for: glob@7.1.7 want: ^1.0.0
91 silly fetch manifest wrappy@1
92 silly fetch manifest brace-expansion@^1.1.7
93 http fetch GET 304 https://registry.npmjs.org/brace-expansion 53ms (from cache)
94 http fetch GET 304 https://registry.npmjs.org/wrappy 58ms (from cache)
95 timing idealTree:node_modules/glob Completed in 65ms
96 timing idealTree:node_modules/fs.realpath Completed in 0ms
97 silly placeDep ROOT wrappy@1.0.2 OK for: inflight@1.0.6 want: 1
98 timing idealTree:node_modules/inflight Completed in 1ms
99 timing idealTree:node_modules/inherits Completed in 0ms
100 timing idealTree:node_modules/jasmine-core Completed in 0ms
101 silly placeDep ROOT compute-lcm@1.1.2 OK for: json-schema-merge-allof@0.7.0 want: ^1.1.0
102 silly placeDep ROOT json-schema-compare@0.2.2 OK for: json-schema-merge-allof@0.7.0 want: ^0.2.2
103 silly placeDep ROOT lodash@4.17.21 OK for: json-schema-merge-allof@0.7.0 want: ^4.17.4
104 silly fetch manifest compute-gcd@^1.2.1
105 silly fetch manifest validate.io-array@^1.0.3
106 silly fetch manifest validate.io-function@^1.0.2
107 silly fetch manifest validate.io-integer-array@^1.0.0
108 http fetch GET 304 https://registry.npmjs.org/compute-gcd 52ms (from cache)
109 http fetch GET 304 https://registry.npmjs.org/validate.io-function 58ms (from cache)
110 http fetch GET 304 https://registry.npmjs.org/validate.io-integer-array 58ms (from cache)
111 http fetch GET 304 https://registry.npmjs.org/validate.io-array 67ms (from cache)
112 timing idealTree:node_modules/json-schema-merge-allof Completed in 71ms
113 silly placeDep ROOT compute-gcd@1.2.1 OK for: compute-lcm@1.1.2 want: ^1.2.1
114 silly placeDep ROOT validate.io-array@1.0.6 OK for: compute-lcm@1.1.2 want: ^1.0.3
115 silly placeDep ROOT validate.io-function@1.0.2 OK for: compute-lcm@1.1.2 want: ^1.0.2
116 silly placeDep ROOT validate.io-integer-array@1.0.0 OK for: compute-lcm@1.1.2 want: ^1.0.0
117 silly fetch manifest validate.io-integer@^1.0.4
118 http fetch GET 304 https://registry.npmjs.org/validate.io-integer 46ms (from cache)
119 timing idealTree:node_modules/compute-lcm Completed in 50ms
120 timing idealTree:node_modules/compute-gcd Completed in 0ms
121 timing idealTree:node_modules/json-schema-compare Completed in 0ms
122 silly placeDep ROOT @apidevtools/json-schema-ref-parser@8.0.0 OK for: json-schema-ref-parser@8.0.0 want: 8.0.0
123 silly fetch manifest call-me-maybe@^1.0.1
124 silly fetch manifest js-yaml@^3.13.1
125 silly fetch manifest @jsdevtools/ono@^7.1.0
126 http fetch GET 304 https://registry.npmjs.org/js-yaml 53ms (from cache)
127 http fetch GET 304 https://registry.npmjs.org/call-me-maybe 135ms (from cache)
128 http fetch GET 304 https://registry.npmjs.org/@jsdevtools%2fono 172ms (from cache)
129 timing idealTree:node_modules/json-schema-ref-parser Completed in 176ms
130 silly placeDep ROOT @jsdevtools/ono@7.1.3 OK for: @apidevtools/json-schema-ref-parser@8.0.0 want: ^7.1.0
131 silly placeDep ROOT call-me-maybe@1.0.1 OK for: @apidevtools/json-schema-ref-parser@8.0.0 want: ^1.0.1
132 silly placeDep ROOT js-yaml@3.14.1 OK for: @apidevtools/json-schema-ref-parser@8.0.0 want: ^3.13.1
133 silly fetch manifest argparse@^1.0.7
134 silly fetch manifest esprima@^4.0.0
135 http fetch GET 304 https://registry.npmjs.org/esprima 58ms (from cache)
136 http fetch GET 304 https://registry.npmjs.org/argparse 65ms (from cache)
137 timing idealTree:node_modules/@apidevtools/json-schema-ref-parser Completed in 68ms
138 timing idealTree:node_modules/@jsdevtools/ono Completed in 0ms
139 timing idealTree:node_modules/call-me-maybe Completed in 0ms
140 silly placeDep ROOT argparse@1.0.10 OK for: js-yaml@3.14.1 want: ^1.0.7
141 silly placeDep ROOT esprima@4.0.1 OK for: js-yaml@3.14.1 want: ^4.0.0
142 silly fetch manifest sprintf-js@~1.0.2
143 http fetch GET 304 https://registry.npmjs.org/sprintf-js 58ms (from cache)
144 timing idealTree:node_modules/js-yaml Completed in 61ms
145 silly placeDep ROOT sprintf-js@1.0.3 OK for: argparse@1.0.10 want: ~1.0.2
146 timing idealTree:node_modules/argparse Completed in 1ms
147 timing idealTree:node_modules/esprima Completed in 0ms
148 timing idealTree:node_modules/jsonschema Completed in 0ms
149 silly placeDep ROOT ajv@4.11.8 OK for: jsonvalidator@0.1.1 want: ^4.11.3
150 silly placeDep ROOT debug@2.6.9 OK for: jsonvalidator@0.1.1 want: ^2.6.1
151 silly fetch manifest co@^4.6.0
152 silly fetch manifest json-stable-stringify@^1.0.1
153 silly fetch manifest ms@2.0.0
154 http fetch GET 304 https://registry.npmjs.org/co 52ms (from cache)
155 http fetch GET 304 https://registry.npmjs.org/ms 63ms (from cache)
156 http fetch GET 304 https://registry.npmjs.org/json-stable-stringify 101ms (from cache)
157 timing idealTree:node_modules/jsonvalidator Completed in 104ms
158 silly placeDep ROOT co@4.6.0 OK for: ajv@4.11.8 want: ^4.6.0
159 silly placeDep ROOT json-stable-stringify@1.0.1 OK for: ajv@4.11.8 want: ^1.0.1
160 silly fetch manifest jsonify@~0.0.0
161 http fetch GET 304 https://registry.npmjs.org/jsonify 59ms (from cache)
162 timing idealTree:node_modules/ajv Completed in 62ms
163 timing idealTree:node_modules/co Completed in 0ms
164 silly placeDep ROOT ms@2.0.0 OK for: debug@2.6.9 want: 2.0.0
165 timing idealTree:node_modules/debug Completed in 1ms
166 silly placeDep ROOT jsonify@0.0.0 OK for: json-stable-stringify@1.0.1 want: ~0.0.0
167 timing idealTree:node_modules/json-stable-stringify Completed in 1ms
168 timing idealTree:node_modules/jsonify Completed in 0ms
169 timing idealTree:node_modules/lodash Completed in 0ms
170 silly placeDep ROOT brace-expansion@1.1.11 OK for: minimatch@3.0.4 want: ^1.1.7
171 silly fetch manifest balanced-match@^1.0.0
172 silly fetch manifest concat-map@0.0.1
173 http fetch GET 304 https://registry.npmjs.org/balanced-match 53ms (from cache)
174 http fetch GET 304 https://registry.npmjs.org/concat-map 53ms (from cache)
175 timing idealTree:node_modules/minimatch Completed in 56ms
176 silly placeDep ROOT balanced-match@1.0.2 OK for: brace-expansion@1.1.11 want: ^1.0.0
177 silly placeDep ROOT concat-map@0.0.1 OK for: brace-expansion@1.1.11 want: 0.0.1
178 timing idealTree:node_modules/brace-expansion Completed in 1ms
179 timing idealTree:node_modules/balanced-match Completed in 0ms
180 timing idealTree:node_modules/concat-map Completed in 0ms
181 timing idealTree:node_modules/ms Completed in 0ms
182 timing idealTree:node_modules/once Completed in 0ms
183 timing idealTree:node_modules/path-is-absolute Completed in 0ms
184 timing idealTree:node_modules/sprintf-js Completed in 0ms
185 timing idealTree:node_modules/validate.io-array Completed in 0ms
186 timing idealTree:node_modules/validate.io-function Completed in 0ms
187 silly placeDep ROOT validate.io-integer@1.0.5 OK for: validate.io-integer-array@1.0.0 want: ^1.0.4
188 silly fetch manifest validate.io-number@^1.0.3
189 http fetch GET 304 https://registry.npmjs.org/validate.io-number 41ms (from cache)
190 timing idealTree:node_modules/validate.io-integer-array Completed in 47ms
191 silly placeDep ROOT validate.io-number@1.0.3 OK for: validate.io-integer@1.0.5 want: ^1.0.3
192 timing idealTree:node_modules/validate.io-integer Completed in 0ms
193 timing idealTree:node_modules/validate.io-number Completed in 0ms
194 timing idealTree:node_modules/wrappy Completed in 0ms
195 timing idealTree:buildDeps Completed in 2485ms
196 timing idealTree:fixDepFlags Completed in 0ms
197 timing idealTree Completed in 4409ms
198 timing reify:loadTrees Completed in 4410ms
199 timing reify:diffTrees Completed in 2ms
200 silly reify moves {}
201 timing reify:retireShallow Completed in 0ms
202 timing reify:createSparse Completed in 5ms
203 timing reify:loadBundles Completed in 0ms
204 timing reifyNode:node_modules/brace-expansion Completed in 180ms
205 timing reifyNode:node_modules/ms Completed in 181ms
206 timing reifyNode:node_modules/wrappy Completed in 182ms
207 timing reifyNode:node_modules/path-is-absolute Completed in 182ms
208 timing reifyNode:node_modules/once Completed in 183ms
209 timing reifyNode:node_modules/inflight Completed in 183ms
210 timing reifyNode:node_modules/validate.io-number Completed in 184ms
211 timing reifyNode:node_modules/validate.io-integer Completed in 184ms
212 timing reifyNode:node_modules/validate.io-integer-array Completed in 186ms
213 timing reifyNode:node_modules/validate.io-array Completed in 186ms
214 timing reifyNode:node_modules/validate.io-function Completed in 186ms
215 timing reifyNode:node_modules/compute-gcd Completed in 187ms
216 timing reifyNode:node_modules/compute-lcm Completed in 187ms
217 timing reifyNode:node_modules/inherits Completed in 187ms
218 timing reifyNode:node_modules/jsonvalidator Completed in 188ms
219 timing reifyNode:node_modules/co Completed in 188ms
220 timing reifyNode:node_modules/minimatch Completed in 192ms
221 timing reifyNode:node_modules/fs.realpath Completed in 193ms
222 timing reifyNode:node_modules/balanced-match Completed in 193ms
223 timing reifyNode:node_modules/json-schema-ref-parser Completed in 207ms
224 timing reifyNode:node_modules/concat-map Completed in 207ms
225 timing reifyNode:node_modules/call-me-maybe Completed in 208ms
226 timing reifyNode:node_modules/jsonify Completed in 211ms
227 timing reifyNode:node_modules/glob Completed in 214ms
228 timing reifyNode:node_modules/json-schema-compare Completed in 215ms
229 timing reifyNode:node_modules/jsonschema Completed in 217ms
230 timing reifyNode:node_modules/json-stable-stringify Completed in 226ms
231 timing reifyNode:node_modules/debug Completed in 231ms
232 timing reifyNode:node_modules/sprintf-js Completed in 234ms
233 timing reifyNode:node_modules/esprima Completed in 239ms
234 timing reifyNode:node_modules/esm Completed in 239ms
235 timing reifyNode:node_modules/json-schema-merge-allof Completed in 242ms
236 timing reifyNode:node_modules/jasmine Completed in 250ms
237 http fetch POST 200 https://registry.npmjs.org/-/npm/v1/security/advisories/bulk 255ms
238 timing auditReport:getReport Completed in 255ms
239 timing auditReport:init Completed in 0ms
240 timing reify:audit Completed in 256ms
241 timing reifyNode:node_modules/@apidevtools/json-schema-ref-parser Completed in 264ms
242 timing reifyNode:node_modules/argparse Completed in 272ms
243 timing reifyNode:node_modules/@jsdevtools/ono Completed in 289ms
244 timing reifyNode:node_modules/js-yaml Completed in 294ms
245 timing reifyNode:node_modules/jasmine-core Completed in 302ms
246 timing reifyNode:node_modules/ajv Completed in 375ms
247 timing reifyNode:node_modules/lodash Completed in 775ms
248 timing reify:rollback:createSparse Completed in 480ms
249 timing reify:rollback:retireShallow Completed in 1ms
250 timing command:install Completed in 5682ms
251 verbose stack TypeError: Wrong spec type (git) for RemoteFetcher. Supported types: remote
251 verbose stack     at RemoteFetcher.[_assertType] (/Users/codejedi365/.nvm/versions/node/v14.16.0/lib/node_modules/npm/node_modules/pacote/lib/fetcher.js:223:13)
251 verbose stack     at new FetcherBase (/Users/codejedi365/.nvm/versions/node/v14.16.0/lib/node_modules/npm/node_modules/pacote/lib/fetcher.js:61:22)
251 verbose stack     at new RemoteFetcher (/Users/codejedi365/.nvm/versions/node/v14.16.0/lib/node_modules/npm/node_modules/pacote/lib/remote.js:14:5)
251 verbose stack     at /Users/codejedi365/.nvm/versions/node/v14.16.0/lib/node_modules/npm/node_modules/pacote/lib/git.js:229:16
251 verbose stack     at /Users/codejedi365/.nvm/versions/node/v14.16.0/lib/node_modules/npm/node_modules/cacache/lib/util/disposer.js:26:43
252 verbose cwd /Users/codejedi365/Development/workbench/workspace/gitlab-npm-audit-parser
253 verbose Darwin 20.4.0
254 verbose argv "/Users/codejedi365/.nvm/versions/node/v14.16.0/bin/node" "/Users/codejedi365/.nvm/versions/node/v14.16.0/bin/npm" "install" "--save" "gitlab:gitlab-org/security-products/security-report-schemas#semver:^14.0.1"
255 verbose node v14.16.0
256 verbose npm  v7.7.0
257 error Wrong spec type (git) for RemoteFetcher. Supported types: remote
258 verbose exit 1
```

**Comment by ljharb** [comment]
@codejedi365 you're using npm v7.7. Can you try with v7.13?

**Comment by codejedi365** [comment]
@ljharb, good catch, I didn't realize I was so far behind on this computer and forgot to update and test before posting.  Well the problem resurfaced for 7.7.0 but now it is resolved in v7.13.0! Thanks!  I will note that it was oddly slow but it worked nonetheless.

We can keep this closed! Sorry!
